src/controller/AutoCommissioner.cpp

```diff
@@ -102,9 +102,26 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStage(CommissioningStag
     {
         return CommissioningStage::kCleanup;
     }
+    mEndpoint = 0;
     switch (currentStage)
     {
     case CommissioningStage::kSecurePairing:
+        return CommissioningStage::kReadVendorId;
+    case CommissioningStage::kReadVendorId:
+        return CommissioningStage::kReadProductId;
+    case CommissioningStage::kReadProductId:
+        return CommissioningStage::kReadSoftwareVersion;
+    case CommissioningStage::kReadSoftwareVersion:
+        return CommissioningStage::kGetPartsList;
+    case CommissioningStage::kGetPartsList:
+        return CommissioningStage::kCheckEndpointIsCommissionable;
+    case CommissioningStage::kCheckEndpointIsCommissionable:
+        if (mAllEndpoints.numEndpoints > 0)
+        {
+            // Cycle through the list of endpoints from the end, checking for network cluster.
```

Comment:
So this is trying to build a list of all the network clusters in mNetworkEndpoints (unused so far) and then just checking that endpoint 0 is in fact commissionable?

Reply:
yes